runtimes/data-binding.mdx

```diff
@@ -736,10 +738,11 @@ View models can have properties of type view model, allowing for arbitrary nesti
 
 
 # Examples
-
 <Tabs>
     <Tab title="Web">
-    See [this video](https://x.com/rive_app/status/1907115949234467091) for an overview of data binding using the Web API along with this [CodeSandbox example](https://codesandbox.io/p/sandbox/rive-data-binding-quick-start-js-6v2ltn?file=%2Fsrc%2Findex.mjs%3A25%2C35)
+    See [this video](https://youtu.be/G6IWCZ1IG10?t=1243) for an intro to data binding using the Web runtime along with this [CodeSandbox example](https://codesandbox.io/p/sandbox/rive-data-binding-quick-start-js-6v2ltn?file=%2Fsrc%2Findex.mjs%3A25%2C35).
+
+    <YouTube id="G6IWCZ1IG10" timestamp="1245" />
```

Comment:
The timestamp in "this video" is 1243, but the timestamp of the snippet is 1245. Should they be equal?

Reply:
Yes, good catch! Otherwise, we miss the "I'm @lancesnider" part! I just typed it wrong